dynaml/merge_test.go

```diff
@@ -12,7 +12,7 @@ var _ = Describe("merges", func() {
 		It("evaluates to the merged node", func() {
 			referencedNode := IntegerExpr{42}
 
-			expr := MergeExpr{[]string{"foo", "bar"}}
+			expr := MergeExpr{[]string{"foo", "bar"},false,false,false, ""}
```

Comment:
Inline struct initialization without naming fields can be confusing with this many fields, especially when repeating types and values in this specific case. Can you change this to a multi-line initialization with named fields?

Thanks!
Zak + @robdimsdale